viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,24 +2,50 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
-#ifdef COGNITE_COLOR_BY_TREE_INDEX
+const int RenderTypeColor = 1;
```

Comment:
Is there a way to make sure that these two separate "lists" aren't out of sync? Is it even important?

Reply:
Not any that I know of that we can use easily. It is somewhat important, but I think we are stepping into territory of generating GLSL from code in another language. Might be something to look into, but for the time being I think we just need to try to keep these in sync.